src/features/cellnav/test/uiGridCellNavService.spec.js

```diff
@@ -290,8 +290,8 @@ describe('ui.grid.edit uiGridCellNavService', function () {
         grid.scrollTo( null, null );
       });
       $timeout.flush();
-      
+
       expect(args).toEqual( null );
     });
   });
-});
\ No newline at end of file
+});
```

Comment:
The only changes to this file are whitespace changes. Please remove from your PR.